core/src/main/scala/scala/pickling/Macros.scala

```diff
@@ -504,7 +499,7 @@ trait UnpicklerMacros extends Macro with UnpickleMacros with FastTypeTagMacros {
         if (sym.isModuleClass) {
           q"${sym.module}"
         } else if (cir.javaGetInstance) {
-          q"""java.lang.Class.forName(${tpe.toString}).getDeclaredMethod("getInstance").invoke(null)"""
+          q"""_root_.java.lang.Class.forName(${tpe.toString}).getDeclaredMethod("getInstance").invoke(null)"""
```

Comment:
I have know idea why you would generate code to perform Java reflection like this. What prevents us from generating the call to `$tpe.getInstance` directly?